sway-fmt-v2/src/utils/language/expr/collections.rs

```diff
@@ -16,7 +16,10 @@ impl Format for ExprTupleDescriptor {
         formatter: &mut Formatter,
     ) -> Result<(), FormatterError> {
         match self {
-            Self::Nil => {}
+            Self::Nil => {
+                Self::open_parenthesis(formatted_code, formatter)?;
```

Comment:
Just realized we can actually open and close the parenthesis before and after the match statement instead of calling the functions twice, one per match arm